### ISSUE TITLE
use pretrain model but not works well

### ISSUE THREAD

**Issue by WenzhongYan** [faustomorales/keras-ocr#74]
I used the pretrain model but the loss cannot converge， I observing the intermediate process and found that there may be problems with the link-map  , and the effect is not as good as the pretrain model.  Could I ask you for some help？

**Comment by faustomorales** [comment]
Closing due to lack of detail (please see [contribution guidelines](https://github.com/faustomorales/keras-ocr#contributing)) and inactivity.
